packages/components/psammead-podcast-promo/src/__snapshots__/index.test.jsx.snap

```diff
@@ -751,34 +791,17 @@ exports[`Podcast Promo Examples on-page example 1`] = `
   z-index: 1;
 }
 
-.emotion-20:visited .podcast-promo--visited {
+.emotion-16:visited .podcast-promo--visited {
```

Comment:
How come some styles don't have the emotion naming?

Reply:
We manually assign these classes to the component here https://github.com/bbc/psammead/blob/3abe5ce9093f550d3ecfc44ce89d24f74c330355/packages/components/psammead-podcast-promo/src/examples/basic.jsx#L32

And then apply the style to that class based on the relevant psuedoclass (`&:visited`) here
https://github.com/bbc/psammead/blob/8713ebd264d85249405ce74ab0addc5e76ceb4a1/packages/components/psammead-podcast-promo/src/components/card-link.jsx#L17